packages/main/src/components/AnalyticalTable/TableBody/VirtualTableBody.tsx

```diff
@@ -196,6 +201,7 @@ export const VirtualTableBody = (props: VirtualTableBodyProps) => {
         }
         prepareRow(row);
         const rowProps = row.getRowProps();
+        markNavigatedRow(row);
```

Comment:
You are currently calling this method thrice, I would suggest to store the value in a variable and replace subsequent calls with that variable.
```suggestion
        const isNavigatedCell = markNavigatedRow(row);
```